src/frontend/demo/package.json

```diff
@@ -50,7 +50,7 @@
     "typescript": "4.7.3"
   },
   "scripts": {
-    "compile-translations": "./i18n/compile-translations.js",
```

Comment:
I think you should explain in your commit that using this script from core package also compile message from core package into demo translations and it allows translation override.

Reply:
Agreed